lib/Conversion/LLHDToLLVM/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_mlir_conversion_library(MLIRLLHDToLLVM 
+  LLHDToLLVM.cpp
+
+  DEPENDS
+  MLIRLLHDIncGen
+  MLIRLLHDConversionPassIncGen
+
+  LINK_LIBS PUBLIC
+  MLIRLLVMIR
+  MLIRStandardToLLVM
+  MLIRVector
+  MLIRTransforms
+  LLVMCore
+  LLVMSupport
```

Comment:
LLVMSupport is always included.  it's unnecessary to add explicitly.  LLVMCore should be moved to set(LLVM_LINK_COMPONENTS Core)